apps/condo/domains/meter/tasks/importMeters.js

```diff
@@ -140,7 +140,7 @@ async function importMeters (taskId) {
             await MeterReadingsImportTask.update(context, taskId, {
                 ...dvAndSender,
                 status: COMPLETED,
-            })
+            }, 'id')
```

Comment:
`id` is default value

Reply:
Yes, but function signature don't know about it :)